### ISSUE TITLE
Upgrade existing ganglia-web UI

### ISSUE THREAD

**Issue by dvigneshwer** [ganglia/ganglia-web#322]
Need to update the existing ganglia web dashboard to that dashboard which supports the feature of downloading the report in form of CSV.

Tried the following command from the wiki (https://github.com/ganglia/ganglia-web/wiki#Installation) ,

~~~~
wget --no-check-certificate https://sourceforge.net/projects/ganglia/files/ganglia-web/3.7.2/ganglia-web-3.7.2.tar.gz/download
tar -xvzf ganglia-web-3.7.2.tar.gz
nano Makefile
make install
~~~~

Makefile screenshot
![image](https://cloud.githubusercontent.com/assets/5075497/21096291/537adf60-c085-11e6-9858-d8c447b5d9e0.png)

Version of the system is **CentOS 6.2**

Got the following log after the **make install** command

![image](https://cloud.githubusercontent.com/assets/5075497/21096335/89dce6e8-c085-11e6-871d-97e015fe77b1.png)

Need to help me to update the ganglia dashboard without downtime.

**Comment by vvuksan** [comment]
Are you saying it's not working ?

**Comment by dvigneshwer** [comment]
@vvuksan yeah, I tried to update my existing UI to the latest one and the above steps I tried did not help me to achieve it.

**Comment by vvuksan** [comment]
Install seems to be successful. What doesn't appear to be working ?

**Comment by dvigneshwer** [comment]
The web UI running at IP:80 did not get updated with the new version after installation.

**Comment by konsta61** [comment]
Did you do 'make clean' & 'make' & 'make install' after editing Makefile?
